Framework/Core/include/Framework/RawDeviceSourceHelpers.h

```diff
@@ -0,0 +1,14 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+namespace o2 {
+namespace framework {
```

Comment:
This is empty ?

Reply:
At some point I was thinking about having things like `o2DMAdaptor` as methods in a `RawDeviceSourceHelpers` class, then I changed my mind. Maybe I should change it back actually...